src/main/scala/edu/ie3/simona/model/participant/evcs/EvModelWrapper.scala

```diff
@@ -32,9 +33,11 @@ final case class EvModelWrapper(
 
   def uuid: UUID = original.getUuid
   def id: String = original.getId
-  lazy val sRatedAc: squants.Power =
-    Kilowatts(original.getSRatedAC.to(KILOWATT).getValue.doubleValue)
-  lazy val sRatedDc: squants.Power =
+  lazy val pRatedAc: Power =
+    Kilowatts(
+      original.getSRatedAC.to(KILOWATT).getValue.doubleValue
+    )
+  lazy val pRatedDc: squants.Power =
```

Comment:
Minor point, but this could be `Power` as well. Maybe you'd want to import `squants.Energy` as well to give this a unified look